src/core/autofill.cc

```diff
@@ -219,8 +221,17 @@ AutoFill::Create(
   // Unable to determine the platform so just use the simple autofill,
   // or null if that fails.
   {
+    if (platform == Platform::PLATFORM_UNKNOWN) {
+      LOG_WARNING << "Autofiller failed to detect the platform (verify "
+                     "contents of model directory or use --log-verbose=1 for "
+                     "more details)";
+    } else {
+      LOG_WARNING << "Autofiller failed to retrieve model. Error Details: "
+                  << status.AsString();
```

Comment:
Isn't this duplicated with the verbose message above? i.e. line 212

Reply:
L213 prints the message in verbose mode only. This will let the user know that autofiller failed as a warning.

Reply:
You should print the TRT Autofill status only if there is an error. I don't think it is necessary in the case of success

Reply:
Please note that control will not reach here if there was a success.

Reply:
What @GuanLuo and I are saying is that In 213 you are printing the Status already. When there is a failure it will be logged twice.